src/ui/components/shared/SharingModal/PrivacyDropdown.tsx

```diff
@@ -46,6 +46,30 @@ function DropdownButton({ disabled, children }: { disabled?: boolean; children:
   );
 }
 
+function filterByWorkspaceType(recording: Recording, workspace: Workspace) {
+  if (workspace.isTest && recording.metadata?.test) {
+    return true;
+  } else if (!workspace.isTest && !recording.metadata?.test) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
```suggestion
function filterByWorkspaceType(recording: Recording, workspace: Workspace) {
  return workspace.isTest == recording.metadata?.test;
}
```

Reply:
Mine is definitely more verbose but I did that intentionally for clarity. The proposed version isn't quite right, I think, because it would be loosely comparing a boolean with an object. (or undefined). My original version was similar to that (`!!workspace.isTest === !!recording.metadata?.test`) but I didn't feel it was as clear.

That said, I also just discovered we have `isTest` and `isInTestWorkspace` on recording so I'll use those instead.

Reply:
> The proposed version isn't quite right, I think, because it would be loosely comparing a boolean with an object.

The proposed version should be exactly the same as the more verbose version. Both compare truthy value to a falsy value. One is just a lot more verbose. (IMO this doesn't make it "clearer" in my personal opinion but that's subjective.)